src/pheval/cli_pheval_utils.py

```diff
@@ -11,19 +14,77 @@
     metavar="FILE",
     help="Path to the semantic similarity profile to be scrambled.",
 )
-def scramble_semsim():
+def scramble_semsim(input: click.Path):
     """scramble_semsim"""
     print("running pheval_utils::scramble_semsim command")
 
 
-@click.command()
+@click.command("scramble-phenopackets")
 @click.option(
-    "--input",
-    "-i",
+    "--phenopacket-dir",
+    "-P",
+    metavar="PATH",
+    required=True,
+    help="Path to phenopackets directory",
+)
+@click.option(
+    "--scramble-factor",
+    "-s",
+    metavar=float,
+    required=True,
+    default=0.5,
+    show_default=True,
+    help="Scramble factor for randomising phenopacket phenotypic profiles.",
+    type=float,
+)
+@click.option(
+    "--output-file-suffix",
+    "-o",
+    metavar="<str>",
+    required=True,
+    help="Suffix to append to output file",
+)
+@click.option(
+    "--output-dir",
+    "-O",
+    metavar="PATH",
+    required=True,
+    help="Path for creation of output directory",
+    default="noisy_phenopackets",
+)
+def scramble_phenopackets_command(
+    phenopacket_dir: click.Path,
```

Comment:
I am also not sure on the difference on the difference between click.Path and Path from pathlib - however I have defined all my paths from pathlib and not sure whether the functions will still perform the same when defining it as a click.Path. It would probably be best to stay consistent with defining Path types

Reply:
We should always be using pathlib imo